### ISSUE TITLE
Problem with non-ASCII characters on Windows

### ISSUE THREAD

**Issue by yutannihilation** [Ironholds/WikipediR#9]
I got this error with R 3.3.1 on Windows 10.

```r
WikipediR::page_content("ja", "wikipedia", page_name = "ポケモン")
#> Error in query(url, "pcontent", clean_response, ...) : 
#>   The API returned an error: invalidtitle - Bad title "<U+0192>|<U+0192>P<U+0192><U+201A><U+0192>“"
```

If I convert the parameter for `page_name` into UTF-8 explicitly, it works.

```r
txt <- WikipediR::page_content("ja", "wikipedia", page_name = iconv("ポケモン", to = "UTF-8"))
```

I guess it would be better to pass all arguments to `query` param of `GET()` since `httr` will take care of encodings.

```r
library(httr)

res <- GET("http://en.wikipedia.org/w/api.php",
    query = list(
      format = "json",
      action = "parse",
      prop   = "text|revid",
      page   = "ポケモン")
)
content(res)
#> $parse
#> $parse$title
#> [1] "ポケモン"
#> 
#> $parse$pageid
#> [1] 3211650
#> 
#> $parse$revid
#> [1] 456775067
#> ...
```

**Comment by Ironholds** [comment]
I mean, this is using httr. I'll take a look this evening as to where
encoding is missing.

**Comment by yutannihilation** [comment]
Thanks!

I know you are using httr, but before passing URL to httr, this package encodes URLs [here](https://github.com/Ironholds/WikipediR/blob/master/R/query.R#L19):

```r
 url <- paste0("http://",utils::URLencode(url))
```

We don't need to `URLencode()` by ourselves as `httr` will do it.

```r
library(httr)

url <- "ja.wikipedia.org/w/api.php?format=json&action=parse&prop=text|revid&page=ポケモン"

res <- GET(url)
res
#> Response [https://ja.wikipedia.org/w/api.php?format=json&action=parse&prop=text|revid&page=%e3%83%9d%e3%82%b1%e3%83%a2%e3%83%b3]
#>   Date: 2016-07-18 13:07
#>   Status: 200
#>   Content-Type: application/json; charset=utf-8
#>   Size: 1.14 kB

content(res)$error
#> NULL
```

If we use `URLencode()`, httr will use it as it is.

```r
res <- GET(URLencode(url))
res
#> Response [https://ja.wikipedia.org/w/api.php?format=json&action=parse&prop=text%7Crevid&page=%83%7C%83%50%83%82%83%93]
#>   Date: 2016-07-18 13:08
#>   Status: 200
#>   Content-Type: application/json; charset=utf-8
#>   Size: 174 B

content(res)$error
#> $code
#> [1] "invalidtitle"
#> 
#> $info
#> [1] "Bad title \"ƒ|ƒPƒ‚ƒ“\""
#> 
#> $`*`
#> [1] "See https://ja.wikipedia.org/w/api.php for API usage"
```

The difference is between the `page` query parameters.

w/o `URLencode()`: `page=%e3%83%9d%e3%82%b1%e3%83%a2%e3%83%b3`  
w/ `URLencode()`: `page=%83%7C%83%50%83%82%83%93`

The former is `URLencode()`ed UTF-8 string, while the latter is `URLencode()`ed Shift-JIS string (as Shift-JIS is default encoding in my environment).

So, in short, easy-fix is removing `URLencode()`. But I think it would be better pass all query parameters to httr as list instead of concatenate it by `paste0()`.
